src/main/java/com/sda/games/checkers/model/Game.java

```diff
@@ -103,9 +107,13 @@ public boolean makeMove() throws Exception {
         // Choosing where to move
         while (true) {
             while (true) {
-                System.out.println("Where to go?");
-                endSpotXY = scanner.nextLine();
-
+                do {
```

Comment:
jest taka fajna zasada ktora zazwyczaj sie sprawdza, jak cos musisz skopiowac to znaczy ze to powinno byc funkcja :) i mysle ze tutaj to pasuje idealnie, zamknij to w jakas funkcje :) 
Ogolnie jesli chodzi o te 2 petle to ze ich nie lubie juz wiesz :D ja bym na poczatek postaral sie znalezc czesci wspolne i powynosil je do funkcji. Funkcje sa spoko, opisuja co robi Twoj kod, a jak kogos interesuje to niech se wejdzie do srodka i sprawdzi jak to sie odbywa